models/repo_unit.go

```diff
@@ -145,6 +145,11 @@ func (r *RepoUnit) ExternalTrackerConfig() *ExternalTrackerConfig {
 	return r.Config.(*ExternalTrackerConfig)
 }
 
+// IssueDependenciesConfig returns config for UnitTypeIssueDependencies
+func (r *RepoUnit) IssueDependenciesConfig() *UnitConfig {
```

Comment:
What you use the configuration for?

Reply:
To enable or disable the dependencies in config.